contributing.md

```diff
@@ -18,10 +18,27 @@ As an example, lets say you are refactoring an existing controller and the servi
 2.  Once you reach a point where there is enough definition to create a commit, do so, and then create a PR and invite the team in to review your WIP
 3.  Repeat this process for dependent services
 
-Any code from the old/legacy controller that you feel is useful should be introduced to the new controller after the tests are written and you the SUT starts to emerge.  That way your new controller tests only pass if the code works in the expected way.  
+Any code from the old/legacy controller that you feel is useful should be introduced to the new controller after the tests are written and you the SUT starts to emerge.  That way your new controller tests only pass if the code works in the expected way.
 
 Once the new controller and services are complete, update the routes to point to the new controller, and now you can delete the old controller/components.
 
+## Coding conventions
+
+### General coding style
+- Use object destructuring where possible, export all the functions from a controller to use them as `import { methodName } from './controller';`
```

Comment:
@jon-shipley are we happy with this?  i always had a preference for full controller imports 🤔

Reply:
I think there were issues with some editors that couldn't point to the correct methods when the controller was destructured on the route level.
I believe this is why we decided to avoid destructuring on these cases.
For everything else it should be fine imo.

Reply:
I personally find it easier to understand code that is written as:

```
 someService.someMethod()
```

rather than

```
someMethod()
```

as it is less clear to me where the methods sits.

Reply:
@danwilliams @GeorgeChatzigiannis @jon-shipley can we go with full controller/component import as jon has illustrated above please